sdk/cosmos/azure-cosmos/azure/cosmos/exceptions.py

```diff
@@ -66,8 +66,28 @@ class CosmosAccessConditionFailedError(CosmosHttpResponseError):
 
 
 class CosmosBatchOperationError(HttpResponseError):
-    """A transactional batch request to the Azure Cosmos database service has failed."""
-
+    """A transactional batch request to the Azure Cosmos database service has failed.
+
+    :ivar int error_index: Index of operation within the batch that caused the error.
+    :ivar headers: Error headers.
+    :vartype headers: dict[str, Any]
+    :ivar status_code: HTTP response code.
+    :vartype status_code: int
+    :ivar message: Error message.
+    :vartype message: str
+    :ivar operation_responses: List of failed operations' responses.
+    :vartype operation_responses:
+        Sequence[Union[Tuple[str, Tuple[Any, ...]], Tuple[str, Tuple[Any, ...], Dict[str, Any]]]]
```

Comment:
Having a union of types here is going to be difficult to parse.... Let me double check the code here...

Reply:
From what I can see in the code this will always just be `List[Dict[str, Any]]`?
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py#L2101